doc/jobs/ci_jobs.rst

```diff
@@ -0,0 +1,187 @@
+*CI* jobs
+=========
+
+A *CI* job is used for continuous integration in batch.
+It builds the code and runs the tests to check for regressions.
+It operates on many source repositories as specified in one ore more ``.repos``
+files, and is triggered to produce a nightly "fat" archive, and can also be
```

Comment:
I'd like to get away from using the word "fat" to describe these artifacts. I tried once before to settle on a replacement and then forgot. https://github.com/ros-infrastructure/rep/pull/184 avoids adding a descriptor to the archive altogether. Terms I've considered in the past are "comprehensive" and "omnibus" but I think a "workspace" archive is also reasonable.